signetdev/host/signetdev_linux.c

```diff
@@ -385,14 +386,20 @@ void *transaction_thread(void *arg)
 int signetdev_priv_issue_command(int command, void *p)
 {
 	intptr_t v[2] = {command, (intptr_t)p};
-	write(g_command_pipe[1], v, sizeof(intptr_t) * 2);
+	if (write(g_command_pipe[1], v, sizeof(intptr_t) * 2) == -1) {
+		perror("signetdev_priv_issue_command write error");
+	}
 	char cmd_resp;
-	read(g_command_resp_pipe[0], &cmd_resp, 1);
+	if (read(g_command_resp_pipe[0], &cmd_resp, 1) == -1) {
+		perror("signetdev_priv_issue_command read error");
+	}
 	return cmd_resp;
 }
 
 void signetdev_priv_issue_command_no_resp(int command, void *p)
 {
 	intptr_t v[2] = {command, (intptr_t)p};
-	write(g_command_pipe[1], v, sizeof(intptr_t) * 2);
```

Comment:
Another case where  there should be a return value to caller

Reply:
ok. changed.